content/3.genomics-platform/2.about-our-data/1.data-sets-and-data-access-units.md

```diff
@@ -154,6 +160,12 @@ The raw Data Sets from this initiative are made available via St. Jude Cloud.
 
 The CICERO Data Set contains the samples which were selected for use in the [CICERO Paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02043-x).
 
+### Clinical Research in ALS and Related Disorders for Therapeutic Development Consortium
+
+**DAU**: CReATe | **Tissue Type**: PBMC germline DNA | **Sequencing Type**: WGS | **Samples**: 705
+
+The Phenotype-Genotype-Biomarker (PGB, or PGB1) study (NCT02327845) of the Clinical Research in ALS and Related Disorders for Therapeutic Development (CReATe) Consortium was a natural history and biomarker study of patients with amyotrophic lateral sclerosis (ALS) or a related disorder, including but not limited to ALS-frontotemporal dementia (ALS-FTD), progressive muscular atrophy (PMA), primary lateral sclerosis (PLS), hereditary spastic paraplegia (HSP), and multisystem proteinopathy (MSP). In addition to patients enrolled in the PGB1 Cohort (primary participants), the study also enrolled family members for limited data collection (secondary participants). This dataset includes WGS data from N=705 in PGB1, including N=472 ALS/ALS-FTD, N=20 PMA, N=47 PLS, N=162 HSP, and N=4 with other related disorders.
```

Comment:
We should add a sentence at the end saying "the findings of the project were published in ..."